src/components/Hero/hero.module.scss

```diff
@@ -0,0 +1,35 @@
+@import "../../styles/component";
+
+.hero {
+  background: $yellow;
+  color: $dark-blue;
+  font-weight: 600;
+  line-height: 1.3;
+  margin: 0 auto;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  .imgAndCopy {
```

Comment:
CSS kabob case for the win!

Reply:
Oh sorry, I thought you meant scss file names. Yeah, `styles.imgAndCopy` looks like a className, but it is not. It's a javascript object. Which not only _should_ be camelCase, it probably breaks with dashes. That said, I don't know scss module convention. Lemme know how you want to play this, but my prettier corrects it and  the dashes are not color coded the same.

Reply:
Dashes _would_ break, but _underscores_ don't. I'm indifferent to it, but I have been using underscores.

Reply:
Okay. We can do snake case instead of camel.